types/parse/parse-tests.ts

```diff
@@ -545,17 +470,3 @@ function test_batch_operations() {
     Parse.Object.fetchAll(games, { sessionToken: '' })
     Parse.Object.fetchAllIfNeeded(games, { sessionToken: '' })
 }
-
```

Comment:
@agoldis any particular reason this test was removed. Parse.Query should still be a valid type in the new version correct?

Reply:
are you talking about `test_query_subscribe`? 
> any particular reason this test was removed
The test checks `query.subscribe` functionality, which doesn't exist in the new version.
> Parse.Query should still be a valid type in the new version correct?
Yes, `Parse.Query` is still there, but some methods has gone, e.g. `subscribe`

Reply:
Can you point to me to the documentation that states query.subscribe was deprecated? 
Subscribing is a core feature of live queries in parse server which are basically a wrapper around a web socket implementation.

Reply:
oh 🙀 you're right! good catch! I will fix that and will let you know.

Reply:
Ok great thanks! Can you make sure everything else is there that should be as well? I’m glad you’re getting us upgraded to the latest. Just want to get these few things ironed out and then we can probably get this merged in. Thanks!